packages/react-bindings/src/hooks/useAccessibility.ts

```diff
@@ -62,8 +70,30 @@ const useAccessibility = <Props>(
     actionHandlers,
   )
 
-  return <SlotProps extends Record<string, any>>(slotName: string, slotProps: SlotProps) =>
+  const getA11Props: UseAccessibilityResult = (slotName, slotProps) =>
     mergeProps(slotName, slotProps, definition)
+
+  // Provides an experimental handling for FocusZone definition in behaviors
+  getA11Props.unstable_withFocusZone = (children: React.ReactElement) => {
```

Comment:
It's a bit confusing in this context, what is children and what is element... Let's simplify it a bit.